### ISSUE TITLE
EDMLoss

### ISSUE THREAD

**Issue by chengjwkk** [jhhuangchloe/DiffusionPDE#9]
Thanks for your work! I want to kindly ask why in **EDMLoss**, the default parameters are set to be: **P_mean=-1.2, P_std=1.2, sigma_data=0.5**, cuz you mentioned in another issue that you used Min-Max Normalization to put the data in the range [-1, 1], so why is the **sigma_data** 0.5? Also, are the **P_mean** and **P_std** correspond to this?

**Comment by jhhuangchloe** [comment]
Hello,

Thank you for your feedback! **P_mean**, **P_std**, and **sigma_data** do not correspond to the training data. Please directly refer to [EDM repository](https://github.com/NVlabs/edm/blob/main/training/loss.py#L67) and [EDM paper Table 1](https://arxiv.org/pdf/2206.00364) regarding the **EDMLoss** term. Precisely, these hyperparameters correspond to the noise distribution and loss weighting.

**Comment by chengjwkk** [comment]
Thanks for your clarification!
